src/features/movieList/index.js

```diff
@@ -1,6 +1,7 @@
 import Header from "../../common/Header";
 import MovieTile from "../../common/MovieTile";
-import { StyledHeader, MoviesGrid } from "./styled";
+import { MoviesGrid } from "../../common/MovieTile/styled";
```

Comment:
Miałaś bardzo dobry pomysł z przeniesieniem tego komponentu 👍 Teraz jeszcze myślę, że jak już będziemy mieć API, to można będzie opakować cały index MovieTile w MoviesGrid i wtedy wszędzie wstawiać wygodnie pojedynczy komponent :)